Description: Changed "with" to "width" in ChangeLog.
 Changed the phrase "wide with" to "wide width".
 .
 unifont (1:10.0.01-2) unstable; urgency=medium
 .
   * Update to standards version 4.0.0.
   * Fixed typo in ChangeLog; thanks to Matt Kraai.
Author: Paul Hardy <unifoundry@unifoundry.com>
Origin: Matt Kraai <kraai@ftbfs.com>
Forwarded: not-needed
Last-Update: 2017-06-27

---

--- unifont-10.0.01.orig/ChangeLog
+++ unifont-10.0.01/ChangeLog
@@ -20,7 +20,7 @@
 	  glyphs, to allow for Chinese ideographs U+9FD8..U+9FE9 introduced
 	  in Unicode 10.0.0 to be quadruple-width.  The hexdraw and hex2bdf
 	  utilities already support this, and the TrueType font appears to be
-	  okay, but such wide with use is still experimental.
+	  okay, but such wide width use is still experimental.
 	* font/plane00/unifont-base.hex:
 	  - David Corbett contributed improved glyphs for U+06E5, u+06E6, and
 	    U+FE73 Arabic glyphs.
